packages/listview/src/components/PercentColumn/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useLocale from 'react-rainbow-components/libs/hooks/useLocale';
+import formatPercent from './helpers/formatPercent';
+import { StyledCellContainer } from './styled';
+import { PercentColumnProps } from './types';
+
+const PercentColumn: React.FC<PercentColumnProps> = (props: PercentColumnProps) => {
+    const { value, locale: localeProp, className, style, ...rest } = props;
+    const locale = useLocale(localeProp);
+    const content = formatPercent(value ?? 0, locale, rest);
+
+    return (
+        <StyledCellContainer className={className} style={style}>
+            <span title={content}>{content}</span>
+        </StyledCellContainer>
```

Comment:
I think maybe here we can get rid off the span and put the title directly into the StyledCellContainer